bin/logstash

```diff
@@ -46,4 +46,11 @@ fi
 . "$(cd `dirname $SOURCEPATH`/..; pwd)/bin/logstash.lib.sh"
 setup
 
-ruby_exec "${LOGSTASH_HOME}/lib/bootstrap/environment.rb" "logstash/runner.rb" "$@"
+if [ "$1" = "-V" ] || [ "$1" = "--version" ];
+then
+  LOGSTASH_VERSION_FILE="${LOGSTASH_HOME}/logstash-core/lib/logstash/version.rb"
+  LOGSTASH_VERSION=$(awk '/./{line=$0} END{print line}' $LOGSTASH_VERSION_FILE | cut -d '"' -f 2)
```

Comment:
agree here with @jordansissel concerns, we should be use most cross platform methods possible, my question is why seed and awk, we could also use `grep` and `cut`, right? mostly as you have in line 52, what do you think?